### PR TITLE
Update index.md

### LINKED ISSUE #520: fixed old url for 'sample page'
**Describe the bug**

Hello，I found that MV3 documents still use MV2 sample page，maybe it was an oversight，If so, I will fix it.

**To Reproduce**
1. Go to [overview](https://developer.chrome.com/docs/extensions/mv3/overview/)
2. Scroll down to [Hello extensions](https://developer.chrome.com/docs/extensions/mv3/overview/#hello-extensions)
3. Click on [sample page](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/mv2-archive/tutorials/hello_extensions)

**Expected behavior**
Open the new [sample page](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/examples/hello-world).

**Screenshots**
image

### PR DESCRIPTION
**Pull request by hu-qi**
Fixes #520

Changes proposed in this pull request:

- Update the sample page url

### CODE REVIEW

**Comment by hu-qi** [comment]
@googlebot I signed it!

**Review by dotproto — CHANGES_REQUESTED** [review]
Thanks for this PR! While I appreciate intent here, the Hello World example this page goes through and the one you've linked to are different. I'm going to create a new hello-world extension in the tutorials directory so that they match.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ or by pressing `Ctrl+Shift+F` on your keyboard.
 [4]: https://chrome.google.com/webstore/developer/dashboard
 [5]: https://chrome.google.com/webstore
 [6]: /docs/webstore
-[7]: https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/mv2-archive/tutorials/hello_extensions
+[7]: https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/examples/hello-world
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:123]:
```suggestion
[7]: https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/tutorials/hello-world
```

**Comment by hu-qi** [comment]
@googlebot I fixed it.

**Review by dotproto — APPROVED** [review]

**Comment by dotproto** [comment]
@googlebot I fixed it.

**Comment by hu-qi** [comment]
@googlebot I fixed it.

**Pull request merged**